src/pages/home/home.html

```diff
@@ -12,24 +12,10 @@
     <img src="assets/img/stockpile_tipi.svg" alt="Stockpile logo">
   </div>
 
-  <ion-segment [(ngModel)]="segment">
-    <ion-segment-button [value]="actions.rent">
-      {{actions.rent}}
-    </ion-segment-button>
-    <ion-segment-button [value]="actions.return">
-      {{actions.return}}
-    </ion-segment-button>
-  </ion-segment>
-
-	<ion-item>
-		<ion-input [(ngModel)]="tag" name="tag" type="text" placeholder="Tag">
-		</ion-input>
-	</ion-item>
-
-	<button ion-button class="tag-button" (click)="onNext()" type="submit" block [disabled]="!tag">
-    {{segment}}
+  <button ion-button (click)="onType()" block [outline]="platform.is('cordova')">
```

Comment:
Type comes from typing the barcode. I see how this can be confusing though. I simply followed the same naming convention that I had for `onScan()` when a user clicks on the scan button. Do you it should be renamed?